### ISSUE TITLE
Assertion pnation->player == nullptr failed

### ISSUE THREAD

**Issue by jwrober** [longturn/freeciv21#1173]
**Describe the bug**
```
[critical] freeciv21-client (../common/player.cpp:796) - Assertion pnation->player == nullptr failed
```

**To Reproduce**
1. Run the client from a terminal
2. Load a saved game (preferably one with lots of AI and is late game)
3. See error

**Expected behavior**
No error

**Screenshots**
N/A

**Platform and version (please complete the following information):**
 - OS: Linux
 - Freeciv21 version: 3.0-beta3]
 - Ruleset/Longturn game (if applicable): N/A

**Additional context**
None

**Comment by lmoureaux** [comment]
Possibly fixed in #1175

**Comment by lmoureaux** [comment]
Confirmed fixed in #1175
